sonar-slang-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S122.html

```diff
@@ -0,0 +1,13 @@
+<pre>
+foo(); bar();
```

Comment:
Where's the description?

Reply:
I made a typo on RSPEC (forgot to add noncompliant title)